Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -7170,7 +7175,7 @@ def _implement_retry(self, retries=0,
                 pass
 
         def _http_request(self, method, url_suffix='', full_url=None, headers=None, auth=None, json_data=None,
-                          params=None, data=None, files=None, timeout=10, resp_type='json', ok_codes=None,
+                          params=None, data=None, files=None, timeout=REQUESTS_TIMEOUT, resp_type='json', ok_codes=None,
```

Comment:
Set to None by default. If it is None, then use self.timeout. That way integration can override the value for specific cases.